massa-node/base_config/config.toml

```diff
@@ -33,6 +33,7 @@ version = "DEVE.0.0"
     initial_ledger_path = "base_config/initial_ledger.json"
     operation_batch_size = 1024
     max_send_wait = 500
+    max_gas_per_block = 1000
```

Comment:
I guess we do not want this to be a user-defined config parameter right?

Reply:
right, otherwise a block may or may not be valid depending on the user